modules/markup/markdown/transform_list.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
```

Comment:
checkboxlist is a better name?

Reply:
It matches the node type

![image](https://github.com/go-gitea/gitea/assets/2114189/767c2599-4e43-4194-9e2b-718ea3a887dd)